src/asciidoctor-kroki.js

```diff
@@ -176,7 +176,8 @@ function diagramBlockMacro (name, context) {
       const diagramType = name
       try {
         const diagramText = vfs.read(target)
-        return processKroki(this, parent, attrs, diagramType, diagramText, context)
+        const diagramDir = target.substr(0, target.lastIndexOf('/') + 1)
```

Comment:
we should probably use https://nodejs.org/api/path.html#path_path_dirname_path

Reply:
Nevermind, I forgot we cannot use it here :(

Reply:
Just out of curiosity. Why can `path.dirname(path)` not be used here?

Reply:
Actually, since we are using Browserify, we could. I believe that `path.dirname` will be replaced by the following implementation:

https://github.com/browserify/path-browserify/blob/872fec31a8bac7b9b43be0e54ef3037e0202c5fb/index.js#L293-L316

But I'm fine with this "simple" implementation. We can reevaluate if someone has any issue in a browser environment.